### PR TITLE
[v2] drop IDs transform

### PR DESCRIPTION
**Pull request by alloy**
This addresses the ID fields for types defined in MP’s local schema, not yet for upstream services like Exchange–which we still transform and [will be addressed separately](https://artsyproduct.atlassian.net/browse/PLATFORM-1448).

(Note, for now we're not running v2 tests, we'll fix the test suite after we have everything up and running.)

### CODE REVIEW

File: _schemaV2.graphql [hunk 1]
```diff
@@ -3247,9 +3247,7 @@ type CitySponsoredContent {
 type Collection implements Node {
   # A globally unique ID.
   id: ID!
-
-  # A slug ID.
-  slug: ID!
+  slug: String!
```
[the remark this answers is not in the document]
Comment (alloy) [reply _schemaV2.graphql]: Oh! Yeah, actually let’s do that 👍

[the remark this answers is not in the document]
Comment (zephraph) [reply _schemaV2.graphql]: Nevermind, my bad. They're all strings?

[the remark this answers is not in the document]
Comment (alloy) [reply _schemaV2.graphql]: They are, but semantics. This type just had an extra `slug` definition of its own, all fixed now 👍

**Comment by zephraph** [comment]
A side thing, but I'm seeing ~7 fields called `slug` that have the `String` type. I don't think these need to be updated in this PR, but I'm just listing them out for posterity's sake. 

Article, HighlightedArticle (which is just Article), City, MarketingCollection.. two others that are actually input types.

**Comment by alloy** [comment]
I can do that now and be done with it, just a sec

**Comment by alloy** [comment]
Actually, no, not going to hurry that, they may need some more think work, so I think this PR is good for now.

**Pull request merged**